be/src/runtime/sender_queue.cpp

```diff
@@ -790,20 +793,18 @@ void DataStreamRecvr::PipelineSenderQueue::short_circuit(const int32_t driver_se
     auto& chunk_queue_state = _chunk_queue_states[driver_sequence];
     auto& metrics = _recvr->_metrics[driver_sequence];
     chunk_queue_state.is_short_circuited.store(true, std::memory_order_relaxed);
-    if (_is_pipeline_level_shuffle) {
-        auto& chunk_queue = _chunk_queues[driver_sequence];
-        ChunkItem item;
-        while (chunk_queue.size_approx() > 0) {
-            if (chunk_queue.try_dequeue(item)) {
-                if (item.closure != nullptr) {
-                    COUNTER_UPDATE(metrics.closure_block_timer, MonotonicNanos() - item.queue_enter_time);
-                    item.closure->Run();
-                    chunk_queue_state.blocked_closure_num--;
-                }
-                --_total_chunks;
-                _recvr->_num_buffered_bytes -= item.chunk_bytes;
-                COUNTER_ADD(metrics.peak_buffer_mem_bytes, item.chunk_bytes);
+    auto& chunk_queue = _chunk_queues[driver_sequence];
+    ChunkItem item;
+    while (chunk_queue.size_approx() > 0) {
+        if (chunk_queue.try_dequeue(item)) {
+            if (item.closure != nullptr) {
+                COUNTER_UPDATE(metrics.closure_block_timer, MonotonicNanos() - item.queue_enter_time);
+                item.closure->Run();
+                chunk_queue_state.blocked_closure_num--;
             }
+            --_total_chunks;
+            _recvr->_num_buffered_bytes -= item.chunk_bytes;
+            COUNTER_ADD(metrics.peak_buffer_mem_bytes, item.chunk_bytes);
```

Comment:
there are same logics in clear_buffer_queues and short_circuit, could we add a new function and put it into the function?e.g. clear_chunk_queue(ChunkQueue&)

Reply:
The short circuit is only used for pipeline level shuffle and it should be kept. And I've reverted the previous change, PTAL again.